lando/k8s/jobmanager.py

```diff
@@ -98,11 +102,16 @@ def create_stage_data_job(self, input_files):
                                 labels=labels)
         return self.cluster_api.create_job(self.names.stage_data, job_spec, labels=labels)
 
-    def _create_stage_data_config_map(self, name, filename, workflow_url, job_order, input_files):
+    def _create_stage_data_config_map(self, name, filename, workflow, input_files):
+        workflow_type = workflow.workflow_type
         items = [
-            self._stage_data_config_item("url", workflow_url, self.names.workflow_path),
-            self._stage_data_config_item("write", job_order, self.names.job_order_path),
+            self._stage_data_config_item("url", workflow.workflow_url, self.names.workflow_download_dest),
         ]
+        if workflow_type == WorkflowTypes.ZIPPED:
+            items.append(self._stage_data_config_item("unzip", self.names.workflow_download_dest, Paths.WORKFLOW))
+        elif workflow_type != WorkflowTypes.PACKED:
```

Comment:
This if/elif construction reads a little awkward. I think it would be cleaner as you've done elsewhere:

```
if zipped:
  items.append()
elif packed:
  pass
else:
  raise()
```